### PR TITLE
Add subscription plan for configuration

### PR DESCRIPTION
**Pull request by phieronymus**
The same configuration has to be added to the private node

### CODE REVIEW

File: src/resources/common/Configuration.ts [hunk 1]
```diff
@@ -169,6 +169,23 @@ export interface InstallmentsConfiguration {
 
 export interface InstallmentsConfigurationItem extends PaymentTypeConfiguration, InstallmentsConfiguration {}
 
+export type SubscriptionsPlanConfiguration = {
```
Comment (gdp88) [inline src/resources/common/Configuration.ts:172]: very minor but if a variable name consists of multiple words I m more used to having them in their singular form and then use plural (if needed) on the last one. So here I would go with `SubscriptionPlanConfiguration`.
But seems like we have the same naming pattern for other `interface` names in this file as well so if we want to keep it consistent maybe we don't need to change 🙇

**Review by gdp88 — APPROVED** [review]

**Comment by gdp88** [comment]
very minor comment..lgtm!

File: src/resources/common/Configuration.ts (hunk 1 above)
Comment (phieronymus) [reply src/resources/common/Configuration.ts:172]: Ups, thanks! That is correct, the type should follow the property name. Thanks for catching it.

Comment (gdp88) [reply src/resources/common/Configuration.ts]: thanx! resolving!

**Comment by gdp88** [comment]
alright merging this one!

**Pull request merged**